addons/sys_list/initSettings.sqf

```diff
@@ -0,0 +1,122 @@
+// Hint Background Color
+[
+    QGVAR(hintBgColor),
+    "COLOR",
+    localize LSTRING(hintBgColor_displayName),
+    "ACRE2 UI",
```

Comment:
Stringtable. Can also copy over CBA strings that are already translated: https://github.com/CBATeam/CBA_A3/blob/master/addons/ui/stringtable.xml#L40